lib/dry/auto_inject/injection.rb

```diff
@@ -0,0 +1,170 @@
+module Dry
+  module AutoInject
+    # @api private
+    class Injection < Module
+      InstanceMethods = Class.new(Module)
```

Comment:
FYI: I added this initially because I used it as a way to check a method's owner back when I was trying the `#super_method` approach. I'm not doing that anymore, so _technically_ this doesn't have to exist, but I like it because it makes things clear to any developer who inspect's a class’s ancestors – they'll see `Dry::AutoInject::Injection::InstanceMethods` rather than just an anonymous module.